### PR TITLE
Kata  Making Dom Stateful

### PR DESCRIPTION
**Pull request by davidskyline**

### CODE REVIEW

File: user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+- To which state the instance can **transition**.
+- What **actions** are allowed.
+- What **buttons** to show on the form.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.3.
+
+Expected duration: 20 minutes
+
+> [!TIP]
+> See also: [Kata #15: Getting started with DataMiner Object Models](https://community.dataminer.services/courses/kata-15/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md:21]: I noticed that this tutorial is for the upcoming Kata 24; will you update this link when the new Kata is available?

**Comment by davidskyline** [comment]
@MariekeGO I reviewed the last version. I made a change to the title, because it is not really the dom module which is made stateful but the objects themselves. I also added the link to the Kata video like you suggested. Thanks for reviewing

File: user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.10 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- An [indexing database](xref:Indexing_Database) or [Storage as a Service](xref:STaaS).
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md:34]: Note that this overview should only contain the steps below the overview, not the prerequisites etc. above it.

File: user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+- Which fields are **mandatory**.
+- To which state the instance can **transition**.
+- What **actions** are allowed.
+- What **buttons** to show on the form.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.3.
+
+Expected duration: 20 minutes
+
+<!-- Uncomment this section when the link to the Kata is available:
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md:20]: As the link to the Kata is not yet available, I've commented out this block for now. As soon as the link is available, we can uncomment it.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**